README.md

```diff
@@ -122,8 +122,9 @@ LIGHTS_PIPE_TOGGLE | Toggle pipelight on and off
 CRUISE_CONTROL_TOGGLE | Toggle cruise control on and off
 DRIVE_DIRECTION_TOGGLE | Toggle vehicle drive direction
 COVER_TOGGLE | Toggle cover state
-ATTACHERJOINT_LIFT_LOWER | Lift/lower implement on attacherJoint index | ".attacherJoint#index"
+ATTACHERJOINT_LIFT_LOWER | Lift/lower implement attached to attacherJoint index | ".attacherJoint#index"
```

Comment:
Edited the description of ATTACHERJOINT_LIFT_LOWER to represent the differences of the functions more clearly